src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -45,6 +45,14 @@ var fluid_2_0 = fluid_2_0 || {};
                 listener: "{that}.applier.modelChanged.addListener",
                 args: ["isShowing", "{that}.refreshView"]
             },
+            "onCreate.setAriaControls":{
```

Comment:
Talked to "clown" in the channel. He suggested a role of group.
https://botbot.me/freenode/fluid-work/2015-10-01/?msg=50937400&page=1